--- nvim/init.lua	2023-01-05 17:35:47.196979300 -0500
+++ /mnt/c/Users/Stephen/AppData/Local/nvim/lua/user/init.lua	2023-01-08 11:17:40.363903500 -0500
@@ -1,4 +1,4 @@
-local cmp = require "cmp"
+-- local cmp = require "cmp"
 
 local config = {
   options = {
@@ -55,18 +55,18 @@
         "kylechui/nvim-surround",
         config = function() require("nvim-surround").setup {} end,
       },
-      {
-        "declancm/cinnamon.nvim",
-        config = function()
-          require('cinnamon').setup {
-            extra_keymaps = true,
-            extended_keymaps = true,
-            override_keymaps = true,
-            always_scroll = true,
-            scroll_limit = 250,
-          }
-        end,
-      },
+      -- {
+      --   "declancm/cinnamon.nvim",
+      --   config = function()
+      --     require('cinnamon').setup {
+      --       extra_keymaps = true,
+      --       extended_keymaps = true,
+      --       override_keymaps = true,
+      --       always_scroll = true,
+      --       scroll_limit = 250,
+      --     }
+      --   end,
+      -- },
       {
         "simrat39/rust-tools.nvim",
         after = "mason-lspconfig.nvim", -- make sure to load after mason-lspconfig
@@ -127,16 +127,36 @@

-    ["mason-lspconfig"] = {
-      ensure_installed = { "rust_analyzer", "tsserver" }, -- install rust_analyzer
-    },
+    -- ["mason-lspconfig"] = {
+    --   ensure_installed = { "rust_analyzer", "tsserver" }, -- install rust_analyzer
+    -- },
+
+    ["toggleterm"] = function(config)
+      config.shell = "nu"
+      return config
+    end,
 
     -- {
     --   "pocco81/auto-save.nvim",
@@ -199,15 +219,15 @@
     -- typescript-language-server
   },
 
-  cmp.setup {
-    completion = {
-      -- auto select first completion
-      completeopt = 'menu,menuone,noinsert'
-    },
-  },
-  cmp.setup.filetype({ 'markdown' }, {
-    enabled = false,
-  }),
+  -- cmp.setup {
+  --   completion = {
+  --     -- auto select first completion
+  --     completeopt = 'menu,menuone,noinsert'
+  --   },
+  -- },
+  -- cmp.setup.filetype({ 'markdown' }, {
+  --   enabled = false,
+  -- }),
 
   mappings = {
     n = {
@@ -234,22 +254,22 @@
   --     sonokai_style = "shusia"
   --   }
   -- },
-  highlights = {
-    -- apply highlight group to all colorschemes (include the default_theme)
-    init = {
-      -- set the transparency for all of these highlight groups
-      Normal = { bg = "NONE", ctermbg = "NONE" },
-      NormalNC = { bg = "NONE", ctermbg = "NONE" },
-      CursorColumn = { cterm = {}, ctermbg = "NONE", ctermfg = "NONE" },
-      CursorLine = { cterm = {}, ctermbg = "NONE", ctermfg = "NONE" },
-      CursorLineNr = { cterm = {}, ctermbg = "NONE", ctermfg = "NONE" },
-      LineNr = {},
-      SignColumn = {},
-      StatusLine = {},
-      NeoTreeNormal = { bg = "NONE", ctermbg = "NONE" },
-      NeoTreeNormalNC = { bg = "NONE", ctermbg = "NONE" },
-    },
-  },
+  -- highlights = {
+  --   -- apply highlight group to all colorschemes (include the default_theme)
+  --   init = {
+  --     -- set the transparency for all of these highlight groups
+  --     Normal = { bg = "NONE", ctermbg = "NONE" },
+  --     NormalNC = { bg = "NONE", ctermbg = "NONE" },
+  --     CursorColumn = { cterm = {}, ctermbg = "NONE", ctermfg = "NONE" },
+  --     CursorLine = { cterm = {}, ctermbg = "NONE", ctermfg = "NONE" },
+  --     CursorLineNr = { cterm = {}, ctermbg = "NONE", ctermfg = "NONE" },
+  --     LineNr = {},
+  --     SignColumn = {},
+  --     StatusLine = {},
+  --     NeoTreeNormal = { bg = "NONE", ctermbg = "NONE" },
+  --     NeoTreeNormalNC = { bg = "NONE", ctermbg = "NONE" },
+  --   },
+  -- },
 
   -- taken from https://astronvim.github.io/Recipes/alpha#open-alpha-automatically-when-no-more-buffers
   polish = function()
